src/styles/settings/colors/colors.story.js

```diff
@@ -1,10 +1,66 @@
+import { createEventHook } from '@vueuse/core';
 import Color from 'color';
+import {
+  getCurrentInstance, onBeforeMount, onBeforeUnmount,
+} from 'vue';
 
 export default {
   title: 'CSS/Colors',
 };
 
+function useCssResourceAddonSync({ global, document }) {
+  const onChanged = createEventHook();
+
+  let mutationObserver;
+
+  onBeforeMount(() => {
```

Comment:
Why in previous implementation it was in mounted hook and now in beforeMount?

Reply:
True, I'll fix that.